clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -56,6 +57,23 @@ steps:
       action: start
       labelSelector: group = volume-test
       threshold: {{$POD_STARTUP_SLO}}s
+{{ if $DEPLOY_CSI_DRIVER }}
+- phases:
+    - namespaceRange:
+        min: 1
+        max: 1
+      replicasPerNamespace: 1
+      tuningSet: Sequence
+      objectBundle:
+          objectTemplatePath: {{.CSI_DRIVER_PATH}}
+- measurements:
+    - Identifier: WaitForRunningDriverPods
+      Method: WaitForRunningPods
+      Params:
+        desiredPodCount: {{SumInt .Nodes 1}}
```

Comment:
Why N+1 ?

Reply:
One driver per node plus a cluster wide driver.

Reply:
worth commenting in the code here